src/dcd_mapping/vrs_map.py

```diff
@@ -623,6 +623,8 @@ def _construct_vrs_allele(
                 msg = "Must provide sequence id to construct pre-mapped VRS allele"
                 raise ValueError(msg)
             allele.location.sequenceReference.refgetAccession = sequence_id
+        else:
+            allele.location.sequenceReference.label = hgvs_string.split(":")[0]
```

Comment:
There will always be a colon in the hgvs string right? Just making sure we don't ever raise a key error down below.

Reply:
There should always be a colon in the post-mapped hgvs string, as long as the hgvs string was created by the function _create_post_mapped_hgvs_strings.

Reply:
We are assuming that there's no colon in the transcript or chromosome name, though. I think that's a safe assumption, but if that assumption were to be wrong, then the post-mapped variant would not be created for the variant, and you're right that the associated error for that variant would be a KeyError.